src/components/Community/Community.jsx

```diff
@@ -33,6 +33,13 @@ const Community = () => {
             alt="India Linux User Group"
           />
         </a>
+        <a href="https://fossunited.org/" target="_blank" rel="no noreferrer">
+          <img
+            className={`${styles.community__chapter} ${styles.outline}`}
```

Comment:
```suggestion
            className={`${styles.community__chapter}`}
```
this should remove the blur